src/oas2/__tests__/accessors.test.ts

```diff
@@ -87,7 +87,7 @@ describe('accessors', () => {
             },
```

Comment:
Looks good but needs some more tests: something that can prove we support both AND & OR cases.

Reply:
@chris-miaskowski @lag-of-death is there anything left to address here?

Reply:
@lottamus, no, the test was added:
https://github.com/stoplightio/http-spec/pull/15#discussion_r315112668